src/videosubscriberaccount.cs

```diff
@@ -0,0 +1,145 @@
+﻿//
+// VideoSubscriberAccount bindings
+//
+// Authors:
+//	Alex Soto  <alex.soto@xamarin.com>
+//
+// Copyright 2016 Xamarin Inc. All rights reserved.
+//
+
+using System;
+using XamCore.Foundation;
+using XamCore.ObjCRuntime;
+using XamCore.UIKit;
+
+namespace XamCore.VideoSubscriberAccount {
+
+	[Native]
+	[Introduced (PlatformName.iOS, 10, 0)]
+	[Introduced (PlatformName.TvOS, 10, 0)]
+	[Unavailable (PlatformName.WatchOS)]
+	[ErrorDomain ("VSErrorDomain")]
+	public enum VSErrorCode : nint {
+		AccessNotGranted,
+		UnsupportedProvider,
+		UserCancelled,
+		ServiceTemporarilyUnavailable,
+		ProviderRejected
+	}
+
+	[Native]
+	[Introduced (PlatformName.iOS, 10, 0)]
+	[Introduced (PlatformName.TvOS, 10, 0)]
+	[Unavailable (PlatformName.WatchOS)]
+	public enum VSAccountAccessStatus : nuint {
+		NotDetermined,
+		Restricted,
+		Denied,
+		Granted
+	}
+
+	interface IVSAccountManagerDelegate { }
+
+	[Protocol, Model]
+	[Introduced (PlatformName.iOS, 10, 0)]
+	[Introduced (PlatformName.TvOS, 10, 0)]
+	[Unavailable (PlatformName.WatchOS)]
+	[BaseType (typeof (NSObject))]
+	interface VSAccountManagerDelegate {
+
+		[Abstract]
+		[Export ("accountManager:presentViewController:")]
+		void PresentViewController (VSAccountManager accountManager, UIViewController viewController);
+
+		[Abstract]
+		[Export ("accountManager:dismissViewController:")]
+		void DismissViewController (VSAccountManager accountManager, UIViewController viewController);
+	}
+
+	[Introduced (PlatformName.iOS, 10, 0)]
+	[Introduced (PlatformName.TvOS, 10, 0)]
+	[Unavailable (PlatformName.WatchOS)]
+	[BaseType (typeof (NSObject))]
+	interface VSAccountManager {
+
+		[Field ("VSCheckAccessOptionPrompt")]
+		NSString CheckAccessOptionPromptKey { get; }
+
```

Comment:
Yeah I was thinking since it was just a single key adding it here was fine, will move it into another class maybe `VSCheckAccessOptionsKeys`